ibm/data_source_ibm_pi_instance.go

```diff
@@ -180,6 +185,11 @@ func dataSourceIBMPIInstancesRead(ctx context.Context, d *schema.ResourceData, m
 	d.Set("storage_type", powervmdata.StorageType)
 	d.Set("storage_pool", powervmdata.StoragePool)
 	d.Set("license_repository_capacity", powervmdata.LicenseRepositoryCapacity)
+	if len(strings.TrimSpace(*powervmdata.PlacementGroup)) > 0 && *powervmdata.PlacementGroup == "none" {
```

Comment:
You can directly set the value to `placement_group_id`. No need to set `""`.

Reply:
```
	if *powervmdata.PlacementGroup != "none" {
		d.Set("placement_group_id", powervmdata.PlacementGroup)
	}
```

Reply:
Done